site/src/components/SecondaryNavigation/SecondaryNavigation.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import _ from 'lodash';
+
+import { Box, Button, Drawer, useDrawer } from '@sparkpost/matchbox';
+import { ChevronLeft, ChevronRight } from '@sparkpost/matchbox-icons';
+import SideNavigation from '../SideNavigation/SideNavigation';
+
+function SecondaryNavigation(props) {
+  const { navItems } = props;
+
+  const { label: primaryLabel, childRoutes } = _.find(navItems, [
+    'selected',
+    true
+  ]);
```

Comment:
I think the build error is coming from here – maybe provide a default if there are no navitems like `= _.find(navItems, ['selected', true]) || {}`